custom_components/tuya_local/devices/reiga_52_fan_light.yaml

```diff
@@ -0,0 +1,77 @@
+name: Reiga Ceiling Fan
+products:
+  - id: g0ewlb1vmwqljzji
+    name: Reiga 52 Ceiling Fan with Light
+primary_entity:
+  entity: fan
+  translation_key: fan_with_presets
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: normal
+          value: normal
+        - dps_val: nature
+          value: nature
+        - dps_val: sleep
+          value: sleep
+    - id: 3
+      type: integer
+      name: speed
+      range:
+        min: 1
+        max: 6
+      mapping:
+        - scale: 0.06
+    - id: 8
+      type: string
+      name: direction
+secondary_entities:
+  - entity: select
+    name: Timer
+    icon: "mdi:timer"
+    category: config
+    dps:
+      - id: 22
+        type: string
+        name: option
+        mapping:
+          - dps_val: "off"
+            value: "Off"
+          - dps_val: "1hour"
+            value: "1 hour"
+          - dps_val: "2hour"
+            value: "2 hours"
+          - dps_val: "4hour"
+            value: "4 hours"
+          - dps_val: "8hour"
+            value: "8 hours"
+  - entity: light
+    name: Light
+    icon: "mdi:lightbulb"
+    category: config
+    dps:
+      - id: 15
+        type: boolean
+        name: switch
+      - id: 16
+        type: integer
+        name: brightness
+
+        mapping:
```

Comment:
I'm not sure if not having a range here will affect anything.  I have always included it in the past for any numeric value that can be set, but since light brightness has a fixed range of 0-255 in HA, maybe it is OK to leave off.  Keep watching for errors in the log I guess.

Reply:
I added it to be safe. Seems not to have broken anything. Thanks for the tip.